data/campaigns/Heir_To_The_Throne/scenarios/17_Scepter_of_Fire.cfg

```diff
@@ -617,6 +617,51 @@
         {CLEAR_VARIABLE delfador_moves}
     [/event]
 
+    # Introductory dialog to give the player a hint that she may lose units to
+    # {EXPAND_LAVA}.
+    [event]
+        name=turn 2
+        # The noise was played by {EXPAND_LAVA}.
+        [message]
+            id=Konrad
+            message=_ "That noise was made by no troll! Can you tell its source?"
+        [/message]
+        [if]
+            [have_unit]
+                race=dwarf
+            [/have_unit]
+            [then]
+                [message]
+                    race=dwarf
+                    message=_ "Lord Konrad, yon rumble was made by nae enemy o’ us. ’Tis the echoes of the very bedrock witherin’ and crumbling beneath us that we ’eard. Are ye nae feeling the heat redoubling? Is yer breath nae difficult for the air, sowled by ashes? I beseech that we not tarry ’ere any more than we ha’ to, lest the forming abysses claim us."
+                [/message]
+            [/then]
+            [elseif]
+                # At this point, the lava is shrouded. Let's assume that among
+                # mages, only high-level ones know the rumble was caused by
+                # lava, either because they know the Sceptre's lore or because
+                # they are can sense the fire. Remember that level 1 Mages are
+                # apprentices, according to lore.
+                [have_unit]
+                    type=Arch Mage,Great Mage,Silver Mage
+                [/have_unit]
+                [then]
+                    [message]
+                        type=Arch Mage,Great Mage,Silver Mage
+                        message= _ "Prince Konrad, that din was made by no enemy of ours. It was the sound of the very earth that’s boiling around us, of the fire and lava that are breaching this cave even now. We must tread with caution, for the very ground we stand on may turn against us."
+                    [/message]
+                [/then]
+            [/elseif]
+            [else]
+                # Fallback, in case the player doesn't have either dwarves or magi.
```

Comment:
The player should always have Delfador in this scenario (although as an Elder Mage he won't be chosen by the line above). I think he'd be a better choice for fallback than Li'sar.

Reply:
What's your reasoning? The mage's line _could_ be spoken by Delfador, but I like having Li'sar as a fallback, as it shows her acting amicably toward Konrad. It helps build the transition from "I have a score to settle with you once this temporary alliance is over" at the end of S16 to "If you don't cross the Great River, I'll spare your life" at the start of S18.

Reply:
Thanks for the explanation, and I like your reasoning. I'm thinking she should always say this, even if a dwarf or mage has already spoken.

Reply:
That's a fair point. It actually plays rather well even if a dwarf or mage has given their commentary above.

Reply:
Good suggestion. That will be even clearer a hint. Done.